kv/txn_coord_sender.go

```diff
@@ -54,22 +55,27 @@ import (
 // periodically on their own.
 type txnMetadata struct {
 	// txn is the transaction struct from the initial AddRequest call.
-	txn proto.Transaction
```

Comment:
why this change? I'd avoid a pointer here. What would a txnMetadata without a Txn be good for?